openstudiocore/src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -174,31 +185,37 @@ namespace openstudio{
       if (m_connectionOpen)
       {
         try {
-          execAndThrowOnError("CREATE INDEX rmdTI ON ReportMeterData (TimeIndex ASC);");

```

Comment:
I don't know if they will fail but they definitely won't all work on E+ 8.2 and earlier